### PR TITLE
Test templates in CI

### PR DESCRIPTION
**Pull request by 72636c**

### CODE REVIEW

**Comment by 72636c** [comment]
Requires https://github.com/seek-oss/skuba-dive/pull/4

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -23,10 +22,67 @@ jobs:
...
+      - name: Set up Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+
+      - name: Test template
+        run: yarn test:template greeter
+
+  koa-rest-api:
+    name: koa-rest-api
```
Comment (tobyhei) [inline .github/workflows/validate.yml:53]: can each template use an actions matrix? (I've never used GH actions so don't really know what I'm talking about) https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Comment (72636c) [reply .github/workflows/validate.yml:53]: that sounds fancy so I'll check it out

Comment (72636c) [reply .github/workflows/validate.yml]: works a treat 👏

**Pull request merged**